flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/api/BatchTableEnvironment.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.flink.streaming.api.datastream.DataStream
 import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment
 import org.apache.flink.streaming.api.graph.{StreamGraph, StreamGraphGenerator}
 import org.apache.flink.streaming.api.transformations.StreamTransformation
+import org.apache.flink.table.api.PlannerConfigImpl
```

Comment:
If TableEnv is unified, we actually can't see `PlannerConfigImpl`

Reply:
Yes. However, Planner implementation in blink-planner module could see PlannerConfigImpl. We could move parts to Planner implementation.

Reply:
The other solution is to pull up `getParameters` method in `PlannerConfigIml` to `PlannerConfig` interface, then there is no need to cast `PlannerConfig` to `PlannerConfigIml` here.